dictionaries/en_GB/cspell-tools.config.yaml

```diff
@@ -11,6 +11,7 @@ targets:
       - node_modules/@cspell/dict-en-shared/dict/acronyms.txt
       - node_modules/@cspell/dict-en-shared/dict/shared-additional-words.txt
       - node_modules/@cspell/dict-en-shared/dict/shared-additional-words-ise.txt
+      - node_modules/@cspell/dict-en-shared/dict/shared-additional-words-ize.txt
```

Comment:
should the list be added to the following dictionaries:
- en_GB-legacy
- en_GB-MIT

I think en_GB-MIT is a good candidate, but I'm unsure.